src/connector.rs

```diff
@@ -312,8 +312,11 @@ impl EthConnectorContract {
         // Check promise results
         let data0: Vec<u8> = match sdk::promise_result(0) {
             PromiseResult::Successful(x) => x,
-            _ => sdk::panic_utf8(b"ERR_PROMISE_INDEX"),
+            PromiseResult::Failed => sdk::panic_utf8(b"ERR_???"),
```

Comment:
No descriptive enough.

Reply:
I missed this, wrote it while debugging and never went back to it.